app/models/admin/collection.rb

```diff
@@ -292,6 +292,10 @@ def create_s3_dropbox_directory!
         obj = FileLocator::S3File.new(base_uri.join(n).to_s + '/').object
         obj.exists?
       end
+      avoid_special_chars = ['{', '}', '^', '#', '%', '`', '[', ']', '!', "'", '"', '>', '<', '~', '|']
```

Comment:
@SumithBaddam Do you think using a regex like in this [line](https://github.com/avalonmediasystem/avalon/blob/6c54fe5736ac7aeafe20d1cd7a14a92829bc2a0e/app/models/admin/collection.rb#L267) instead of this array would look more clean?

Reply:
Something like this;
``name.gsub(/[{}^#`%\[\]!'"><~|]/,  '_')``